turtlestar-evaluationtest.py

```diff
@@ -0,0 +1,68 @@
+import pytest
```

Comment:
This test should go into the [`test/test_serializers`](https://github.com/RDFLib/rdflib/tree/main/test/test_serializers)  directory.